packages/cli/src/__tests__/commands/__snapshots__/Version.test.ts.snap

```diff
@@ -38,6 +40,18 @@ Studio                  : STUDIO_VERSION
 
 exports[`version version with custom binaries 1`] = `
 prisma                : 0.0.0
+@prisma/client        : 0.0.0
+Current platform      : TEST_PLATFORM
+Query Engine (Binary) : query-engine ENGINE_VERSION (at sanitized_path/query-engine-TEST_PLATFORM, resolved by PRISMA_QUERY_ENGINE_BINARY)
+Migration Engine      : migration-engine-cli ENGINE_VERSION (at sanitized_path/migration-engine-TEST_PLATFORM, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
+Introspection Engine  : introspection-core ENGINE_VERSION (at sanitized_path/introspection-engine-TEST_PLATFORM, resolved by PRISMA_INTROSPECTION_ENGINE_BINARY)
+Format Binary         : prisma-fmt ENGINE_VERSION (at sanitized_path/prisma-fmt-TEST_PLATFORM, resolved by PRISMA_FMT_BINARY)
+Default Engines Hash  : ENGINE_VERSION
+Studio                : STUDIO_VERSION
+`;
+
+exports[`version version with custom binaries 2`] = `
+prisma                : 0.0.0
```

Comment:
This is weird, only one snapshot is expected for me here 🤔 
(as there is only one expect `expect(cleanSnapshot(data.stdout, version)).toMatchSnapshot()` for the `version with custom binaries` test)

Reply:
Maybe one more snapshot update round is needed, seems that some tests are failing around Version tests